terraform/main.tf

```diff
@@ -23,41 +23,80 @@ resource "google_compute_subnetwork" "honeypot_subnet" {
   network       = google_compute_network.main.self_link
 }
 
-resource "google_compute_firewall" "allow_ssh_in" {
-  name    = "allow-ssh-in"
+# standard firewall settings
+resource "google_compute_firewall" "firewall_standard_rule" {
+  name    = "firewall-standard-rule"
   network = google_compute_network.main.self_link
 
   allow {
     protocol = "icmp"
   }
-
   allow {
     protocol = "tcp"
-    ports    = ["22", "9091"]
+    ports    = ["22"]
```

Comment:
Open port 22 for all VM (for SSH), so that it's easier to control
TODO Later: open port 22 **only** for gateway-vm

Reply:
We can finalize firewall rules in #5 after everything works